Reply/app/src/main/res/layout/fragment_compose.xml

```diff
@@ -141,6 +145,7 @@
 
                     <com.google.android.material.chip.ChipGroup
                         android:id="@+id/recipient_chip_group"
+                        android:onClick="@{() -> handlers.containerTransformChipToCard()"
```

Comment:
i feel like we don't need this click listener applied to the `ChipGroup` if each `Chip` has a click listener that triggers the transform